src/tests/unit/tests/common/components/selector-input-list.test.tsx

```diff
@@ -57,9 +58,9 @@ describe('SelectorInputListTest', () => {
             onDeleteSelector: null,
             onChangeInspectMode: null,
         };
-        const result = Enzyme.shallow(<SelectorInputList {...props} />);
-        genericRenderTests(result, props);
-        expect(result.find('.test-paragraph').getElement).toBeDefined();
+        const renderResult = render(<SelectorInputList {...props} />);
+        genericRenderTests(renderResult, props);
+        expect(renderResult.container.querySelector('.test-paragraph').getAttribute).toBeDefined();
```

Comment:
Because the original test is checking to make sure that the test paragraph exists, we can use `.getByText` and make sure it is not null.
```suggestion
        expect(renderResult.getByText('Test')).not.toBeNull()
```

Reply:
Did the suggested change